docs/pages/authentication.mdx

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
Love this page!

I think we should add a section: `how to choose between ID vs Access tokens`.

Access tokens method requires less work to set up, but it has some limitations (the rooms need to be organized in a tree).

Reply:
Great idea!